src/raft.rs

```diff
@@ -835,22 +818,8 @@ impl<T: Storage> Raft<T> {
     }
 
     /// Sets the vote of `id` to `vote`.
-    ///
-    /// Returns the number of votes for the `id` currently.
-    fn poll(&mut self, id: u64, msg_type: MessageType, vote: bool) -> usize {
-        if vote {
-            info!(
-                "{} received {:?} from {} at term {}",
-                self.tag, msg_type, id, self.term
-            )
-        } else {
-            info!(
-                "{} received {:?} rejection from {} at term {}",
-                self.tag, msg_type, id, self.term
-            )
-        }
+    fn poll(&mut self, id: u64, vote: bool) {
```

Comment:
Some code here was moved to `candidacy_status`

Reply:
Personally I think `poll` is not a suitable name any more.

Reply:
I do too... How about `register_vote`?